template.yml.erb

```diff
@@ -693,6 +696,91 @@ Resources:
                 Period: 300
                 Stat: Sum
 <%end -%>
+# We use shortened versions of names for partition keys (eg, user_id),
+# but values will be a concatenation of the domain name and appropriate ID.
+# Values will look something like:
+# studio.code.org#123456 (user_requests table)
+# studio.code.org#UserId#123456 (blocked_users table)
+<%
+  DOMAIN_AND_USER_ID_COMPOSITE_ATTRIBUTE_NAME = 'user_id'
+  DOMAIN_AND_TOKEN_ID_COMPOSITE_ATTRIBUTE_NAME = 'token_id'
+  DOMAIN_AND_SECTION_OWNER_ID_COMPOSITE_ATTRIBUTE_NAME = 'section_owner_id'
+  ISSUED_AT_TIMESTAMP_ATTRIBUTE_NAME = 'issued_at'
+  TIME_TO_LIVE_ATTRIBUTE_NAME = 'ttl'
+  READ_CAPACITY_PER_SECOND = 1000
+  WRITE_CAPACITY_PER_SECOND = 1000
+-%>
+  BlockedUsersTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: !Sub "${SubDomainName}_blocked_users"
+      KeySchema:
+        - AttributeName: <%=DOMAIN_AND_USER_ID_COMPOSITE_ATTRIBUTE_NAME%>
+          KeyType: HASH
+      ProvisionedThroughput:
+        ReadCapacityUnits: <%=READ_CAPACITY_PER_SECOND%>
+        WriteCapacityUnits: <%=WRITE_CAPACITY_PER_SECOND%>
+      AttributeDefinitions:
+        - AttributeName: <%=DOMAIN_AND_USER_ID_COMPOSITE_ATTRIBUTE_NAME%>
+          AttributeType: S
+      PointInTimeRecoverySpecification:
+        PointInTimeRecoveryEnabled: true
+  TokenStatusTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: !Sub "${SubDomainName}_tokens"
+      KeySchema:
+        - AttributeName: <%=DOMAIN_AND_TOKEN_ID_COMPOSITE_ATTRIBUTE_NAME%>
+          KeyType: HASH
+      ProvisionedThroughput:
+        ReadCapacityUnits: <%=READ_CAPACITY_PER_SECOND%>
+        WriteCapacityUnits: <%=WRITE_CAPACITY_PER_SECOND%>
+      AttributeDefinitions:
+        - AttributeName: <%=DOMAIN_AND_TOKEN_ID_COMPOSITE_ATTRIBUTE_NAME%>
+          AttributeType: S
+      TimeToLiveSpecification:
+        AttributeName: <%=TIME_TO_LIVE_ATTRIBUTE_NAME%>
+        Enabled: true
+  UserRequestsTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: !Sub "${SubDomainName}_user_requests"
+      KeySchema:
+        - AttributeName: <%=DOMAIN_AND_TOKEN_ID_COMPOSITE_ATTRIBUTE_NAME%>
```

Comment:
Was this supposed to be `DOMAIN_AND_USER_ID_COMPOSITE_ATTRIBUTE_NAME`

Reply:
Yes it is 🤦 , thanks for looking closely. Updating now.

Reply:
Done.